README.md

```diff
@@ -38,7 +38,7 @@ docker run \
   lancachenet/monolithic:latest
 ```
 
-Unlike lancachenet/generic this service will cache all cdn services (defined in the [uklans cache-domains repo](https://github.com/uklans/cache-domains) so multiple instances are not required
+Unlike lancachenet/generic this service will cache all cdn services defined in the [uklans cache-domains repo](https://github.com/uklans/cache-domains) so multiple instances are not required.
```

Comment:
If we remove the opening bracket we need to remove the closing one as well

Reply:
Unless I'm mistaken it was an opening bracket that was missing a closing bracket, let me know if I'm wrong

Reply:
I blame mobile view, good point